aws-s3/latest/config/direct-access.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Direct Access URLs
+---
+
+The main purpose of Direct Access URLs (or DAUs) is to accelerate the local download of content by allowing you to have direct content URLs for your binary content that can help with distributed content repositories in customer environments, and cloud deployments.
+
+AWS S3 provides a way of generating [pre-signed URLs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html){:target="_blank"} for sharing objects. This feature is a perfect candidate for implementing direct access to your content, for example, short-cutting the Shared File Store for transformations or faster/direct access in the context of Alfresco Digital Workspace (ADW) to documents).
```

Comment:
>  This feature is a perfect candidate for implementing direct access to your content, for example, short-cutting the Shared File Store for transformations or faster/direct access in the context of Alfresco Digital Workspace (ADW) to documents).

Remove reference to SFS (since currently not implemented or planned - yet) ?

Note: example via ADW is currently work-in-progress,.